doc/contributor/howto-guide-setup-cmake-environment.md

```diff
@@ -60,7 +60,8 @@ to the `cmake` configure step:
 
 ```shell
 cd $HOME/google-cloud-cpp
-cmake -H. -Bcmake-out/home -DCMAKE_TOOLCHAIN_FILE=$HOME/vcpkg/scripts/buildsystems/vcpkg.cmake
+cmake -H. -GNinja -Bcmake-out/home \
```

Comment:
Since we wrote this, most CI scripts write to `build-out/*`, maybe do that here too?

Reply:
Done